### PR TITLE
ss-212: update wallet details page

### PR DESCRIPTION
**Pull request by anton-otroshchenko**

### CODE REVIEW

**Comment by anton-otroshchenko** [comment]
image
Transactions will be added when they will be merged. And also modal component should be added

File: frontend/src/bundles/common/components/header/header.tsx [hunk 1]
```diff
@@ -65,7 +66,7 @@ const Header: React.FC<Properties> = ({
                             pathname === AppRoute.BUDGETS) && (
                             <Tabs tabsData={dataTabs.dashboard} />
                         )}
-                        {pathname === AppRoute.WALLETS && (
+                        {pathname === `/wallet/${id}` && (
```
Comment (VItaliy-Kalenichenko) [inline frontend/src/bundles/common/components/header/header.tsx:69]:
```suggestion
                        {pathname === `${AppRoute.WALLET}/${id}` && (
```

**Review by nikita-remeslov — APPROVED** [review]

**Review by nikita-remeslov — COMMENTED** [review]
In general look good

File: frontend/src/bundles/common/pages/wallet-details/wallet-details.tsx [hunk 2]
```diff
@@ -1,73 +1,171 @@
...
+    },
+];
+
 const WalletDetails: React.FC = () => {
     const { control, errors } = useAppForm<{ note: string }>({
         //It needs to change
         defaultValues: DEFAULT_INPUT,
     });
 
+    const [selectedPeopleMulti, setPeopleSelectedMulti] = useState<
```
Comment (VItaliy-Kalenichenko) [inline frontend/src/bundles/common/pages/wallet-details/wallet-details.tsx:94]: You should come up with the better name
Same for 98 line

File: frontend/src/bundles/common/pages/wallet-details/wallet-details.tsx [hunk 3]
```diff
@@ -1,73 +1,171 @@
...
+                setCategoriesSelectedMulti([]);
+            } else {
+                setCategoriesSelectedMulti(selectedOption);
+            }
+        },
+        [],
+    );
+
+    const handleSliderChange = useCallback(
+        (range: { min: number; max: number }): void => {
```
Comment (VItaliy-Kalenichenko) [inline frontend/src/bundles/common/pages/wallet-details/wallet-details.tsx:129]: Check if there is interface for `{ min: number; max: number }`, if not, create it

File: frontend/src/bundles/common/pages/wallet-details/wallet-details.tsx
Comment (VItaliy-Kalenichenko) [inline frontend/src/bundles/common/pages/wallet-details/wallet-details.tsx:1]: Is all that hard coded data for today's demo?

File: frontend/src/bundles/common/components/header/header.tsx (hunk 1 above)
Comment (anton-otroshchenko) [reply frontend/src/bundles/common/components/header/header.tsx]: fixed

File: frontend/src/bundles/common/pages/wallet-details/wallet-details.tsx (hunk 2 above)
Comment (anton-otroshchenko) [reply frontend/src/bundles/common/pages/wallet-details/wallet-details.tsx]: fixed

File: frontend/src/bundles/common/pages/wallet-details/wallet-details.tsx (hunk 3 above)
Comment (anton-otroshchenko) [reply frontend/src/bundles/common/pages/wallet-details/wallet-details.tsx]: fixed

File: frontend/src/bundles/common/pages/wallet-details/wallet-details.tsx
Comment (anton-otroshchenko) [reply frontend/src/bundles/common/pages/wallet-details/wallet-details.tsx:1]: Yes, for now.

File: frontend/src/bundles/common/pages/wallet-details/wallet-details.tsx [hunk 4]
```diff
@@ -1,79 +1,188 @@
...
-                            >
-                                Future
-                            </Button>
-                            <Button
-                                variant={ButtonVariant.SECONDARY}
+                                variant={ButtonVariant.PRIMARY}
                                 size={ButtonSize.MEDIUM}
                             >
-                                Import
+                                <span>+</span>
```
Comment (teslmik) [inline frontend/src/bundles/common/pages/wallet-details/wallet-details.tsx:158]: May have used the plus icon from the library?
Code_1BBU9SSKho

Comment (anton-otroshchenko) [reply frontend/src/bundles/common/pages/wallet-details/wallet-details.tsx]: fixed

File: frontend/src/bundles/common/pages/wallet-details/styles.module.scss [hunk 5]
```diff
@@ -1,90 +1,50 @@
...
     background: var(--color-blue-100);
 }
 
 .bodyContainer {
     display: flex;
     flex-direction: column;
     align-items: center;
-    max-width: 1293px;
+    max-width: 1192px;
 }
```
[the remark this answers is not in the document]
Comment (anton-otroshchenko) [reply frontend/src/bundles/common/pages/wallet-details/styles.module.scss]: fixed

**Comment by VItaliy-Kalenichenko** [comment]
Resolve conflicts

**Review by nikita-remeslov — APPROVED** [review]

**Pull request merged**